src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -232,3 +233,40 @@ def put(self, request, organization):
             organization.id,
             search_fn,
         )
+
+    def delete(self, request, organization):
+        """
+        Bulk Remove a List of Issues
+        ````````````````````````````
+
+        Permanently remove the given issues. The list of issues to
+        modify is given through the `id` query parameter.  It is repeated
+        for each issue that should be removed.
+
+        Only queries by 'id' are accepted.
+
+        If any ids are out of scope this operation will succeed without
+        any data mutation.
+
+        :qparam int id: a list of IDs of the issues to be removed.  This
+                        parameter shall be repeated for each issue.
+        :pparam string organization_slug: the slug of the organization the
+                                          issues belong to.
+        :auth: required
+        """
+        projects = self.get_projects(request, organization)
+
+        search_fn = functools.partial(
+            self._search, request, organization, projects,
+            self.get_environments(request, organization), {
+                'limit': 1000,
```

Comment:
Wondering if it makes sense to pass these limits to this function inside of delete groups, so that it controls how many items we delete at a time, rather than the endpoint.